applications/StructuralMechanicsApplication/custom_constitutive/small_strain_isotropic_damage_plane_strain_2d.cpp

```diff
@@ -1,100 +1,100 @@
-    rFeatures.mStrainMeasures.push_back(StrainMeasure_Infinitesimal);
-    rFeatures.mStrainSize = this->WorkingSpaceDimension();
-    rFeatures.mSpaceDimension = this->GetStrainSize();
-}
-
-//************************************************************************************
-//************************************************************************************
-
-void LinearIsotropicDamagePlaneStrain2D::save(Serializer& rSerializer) const
-{
-    KRATOS_SERIALIZE_SAVE_BASE_CLASS(rSerializer, LinearIsotropicDamage3D);
-}
-
-//************************************************************************************
-//************************************************************************************
-
-void LinearIsotropicDamagePlaneStrain2D::load(Serializer& rSerializer)
-{
-    KRATOS_SERIALIZE_LOAD_BASE_CLASS(rSerializer, LinearIsotropicDamage3D);
-}
-
-} /* namespace Kratos.*/
+// KRATOS  ___|  |                   |                   |
+//       \___ \  __|  __| |   |  __| __| |   |  __| _` | |
+//             | |   |    |   | (    |   |   | |   (   | |
+//       _____/ \__|_|   \__,_|\___|\__|\__,_|_|  \__,_|_| MECHANICS
+//
+//  License:		 BSD License
+//					 license: structural_mechanics_application/license.txt
+//
+//  Main authors:    Marcelo Raschi
+
+#include "small_strain_isotropic_damage_plane_strain_2d.h"
+#include "structural_mechanics_application_variables.h"
+
+namespace Kratos
+{
+//******************************CONSTRUCTOR*******************************************
+//************************************************************************************
+
+SmallStrainIsotropicDamagePlaneStrain2D::SmallStrainIsotropicDamagePlaneStrain2D()
+    : SmallStrainIsotropicDamage3D()
+{
+}
+
+//********************************COPY CONSTRUCTOR************************************
+//************************************************************************************
+
+SmallStrainIsotropicDamagePlaneStrain2D::SmallStrainIsotropicDamagePlaneStrain2D(
+    const SmallStrainIsotropicDamagePlaneStrain2D &rOther) = default;
+
+//********************************CLONE***********************************************
+//************************************************************************************
+
+ConstitutiveLaw::Pointer SmallStrainIsotropicDamagePlaneStrain2D::Clone() const
+{
+    return Kratos::make_shared<SmallStrainIsotropicDamagePlaneStrain2D>(SmallStrainIsotropicDamagePlaneStrain2D(*this));
+}
+
+//********************************DESTRUCTOR******************************************
+//************************************************************************************
+
+SmallStrainIsotropicDamagePlaneStrain2D::~SmallStrainIsotropicDamagePlaneStrain2D() = default;
+
+//************************************************************************************
+//************************************************************************************
+
+void SmallStrainIsotropicDamagePlaneStrain2D::CalculateElasticMatrix(
```

Comment:
I would use the base law instead (deriving from linear elastic). for a different CL

Reply:
Yep. I'll do that in an upcoming PR.